app/src/main/java/co/omisego/omgshop/extensions/APIErrorExt.kt

```diff
@@ -0,0 +1,12 @@
+package co.omisego.omgshop.extensions
+
+import co.omisego.omisego.constant.enums.ErrorCode
+import co.omisego.omisego.model.APIError
+
+/*
+ * OmiseGO
+ *
+ * Created by Phuchit Sirimongkolsathien on 20/6/2018 AD.
+ * Copyright © 2017-2018 OmiseGO. All rights reserved.
+ */
+fun APIError.isAuthError() = this.code == ErrorCode.CLIENT_INVALID_API_KEY || this.code == ErrorCode.USER_ACCESS_TOKEN_NOT_FOUND
```

Comment:
We don't have access_token_not_found anymore, instead it is `user:auth_token_not_found`, you will need to update the SDK

Reply:
Will do that 👍